tests/test_jsonnet.py

```diff
@@ -27,6 +29,15 @@ def test_yaml_dump(self):
         yaml = yaml_dump("{\"key\":\"value\"}")
         self.assertEqual(yaml, "key: value\n")
 
+    def test_parse_yaml(self):
+        """
+            search_paths is also provided as partial
```

Comment:
What exactly is this comment explaining? Can you simplify/reword it?

Reply:
It is explaining why yaml_to_json takes two arguments whereas parse_yaml takes one. So, we also have to pass search_paths. simplifying it.

Reply:
Maybe update the codeblock to
```
""" 
   This tests the yaml_to_json function.
   It converts the yaml file in test_resources/ to a json file in the current_pwd and checks if the output is as expected.
"""
```  
Generic codeblocks that clearly explain what a function does in 1-2 sentences help other developers immensely when reading lots of code.